examples/simple-example.rs

```diff
@@ -352,24 +319,18 @@ fn main() {
         b: Some(b),
     };
 
-    // Arrange the public input. We expose the multiplication result in row 6
+    // Arrange the public input. We expose the multiplication result in row 0
     // of the instance column, so we position it there in our public inputs.
     let mut public_inputs = vec![Fp::zero(); 1 << k];
```

Comment:
This currently triggers an `InstanceTooLarge` error, because we are now limiting instance columns to `usable_rows` (in response to my earlier review).
```suggestion
    let mut public_inputs = vec![c];
```